static/json/datadog_collection.json

```diff
@@ -1611,6 +1611,61 @@
 				}
 			]
 		},
+		{
+			"name": "Logs *BETA*",
+			"item": [
+				{
+					"name": "Send Logs Over HTTP",
+					"request": {
+						"method": "POST",
+						"header": [
+							{
+								"key": "Content-Type",
+								"name": "Content-Type",
+								"value": "application/json",
+								"type": "text"
+							}
+						],
+						"body": {
+							"mode": "raw",
+							"raw": "{  \n   \"exampleLog\":{  \n      \"title\":\"Example Log from JSON submitted via HTTP\",\n      \"msg\":\"message here\",\n      \"statusCode\": 301,\n      \"responseTime\":\"300ms\",\n      \"category\" : \"test\"\n   }\n}"
```

Comment:
Could we make sure that the [naming convention](https://docs.datadoghq.com/logs/processing/attributes_naming_convention/#getting-started) is used in examples? :) 

`message`,  `http.status_code`,  `duration` (in nanoseconds) ?

Reply:
You bet, updating

Reply:
I have no idea if the `raw` part should be escaped as it is but overall the update looks good.